threestudio/models/guidance/stable_diffusion_vsd_guidance.py

```diff
@@ -398,32 +410,63 @@ def get_latents(
             latents = self.encode_images(rgb_BCHW_512)
         return latents
 
-    def forward(
+    def __call__(
```

Comment:
just leaving it as forward is ok

Reply:
right, I forget about this when copy and paste